### PR TITLE
Support for SPV_QCOM_image_processing2

### PR DESCRIPTION
**Pull request by wooyoungqcom**

### CODE REVIEW

File: known_good.json
Comment (arcady-lunarg) [inline known_good.json:1]: This looks like a whitespace-only change, could you revert it?

**Review by arcady-lunarg — CHANGES_REQUESTED** [review]
The code itself looks fine, but your tests aren't being run, you need to add them to the appropriate list of tests in `gtests/Spv.FromFile.cpp` and generate base results that will end up in the `Test/baseResults` directory by running `build/gtests/glslangtests --update-mode` or similar after you have added the tests to the test list and rebuilt.
Also please revert the whitespace change to `known_good.json`.

**Comment by wooyoungqcom** [comment]
Thank you so much for the review.
I hope I made all the changes you mentioned.

**Review by arcady-lunarg — APPROVED** [review]
LGTM, thanks.

**Pull request merged**